src/main/app/src/Components/Landscape/Modals/Item/Item.tsx

```diff
@@ -136,23 +144,24 @@ const Item: React.FC<Props> = ({ useItem, locateItem, fullyQualifiedItemIdentifi
         onClick={() => {
           locateItem(item.fullyQualifiedIdentifier);
         }}
-        className={classes.floatingButton}
       >
         <FilterCenterFocus />
       </IconButton>
     ) : null;
 
   const labels = item ? getLabels(item) : null;
   const extend = small ? (
-    <IconButton onClick={() => setCompact(!compact)} className={classes.floatingButton}>
+    <IconButton onClick={() => setCompact(!compact)}>
```

Comment:
Maybe this is what breaks the image in the sidebar?

Reply:
fixed